tests_config/run_spec.js

```diff
@@ -9,6 +9,7 @@ const normalizeOptions = require("prettier/src/main/options").normalize;
 const AST_COMPARE = process.env["AST_COMPARE"];
 
 function run_spec(dirname, parsers, options) {
+  global.isInTest = true;
   options = Object.assign(
```

Comment:
Did you try passing `__isInTest: true` in `options` instead? If it triggers a warning we can go with this approach.

Reply:
Hm, I just tried your suggestion and I got:
```
    console.warn node_modules/prettier/node_modules/jest-validate/build/utils.js:44
      ● Validation Warning:

        Unknown option "__isInTest" with value true was found.
        This is probably a typing mistake. Fixing it will remove this message.
```
If this waits for #25 then we could set `__isInTest` as one of the options that this plugin expects :)

Reply:
Just rebased this and changed it to use `__isInTest` :)